src/components/InputInfoButton/index.js

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+
+import InputInfoForm from '../InputInfoForm/index';
+import ShowName from '../ShowName/index';
+import ShowPicture from '../ShowPicture/index';
+
+export default function InputInfoButton() {
+	const [showForm, setShowForm] = useState(false);
+	const [submittedItems, setSubmittedItems] = useState([]);
+
+	const addNewInfo = prevItem => {
+		const newInfoItems = [...submittedItems, prevItem];
+		setSubmittedItems(newInfoItems);
+	};
+
+	return (
+		<>
+			<button onClick={() => setShowForm(!showForm)}>Show Form</button>
+			{showForm && <InputInfoForm addNewInfo={addNewInfo} />}
+			{submittedItems.map((item, index, array) =>
+				index === array.length - 1 ? (
```

Comment:
What was your intention to check if index === array.length - 1?

```suggestion
				submittedItems && submittedItems.map((item, index) => (
					<section key={item.id}>
						<ShowName styledName={item.enteredName}></ShowName>
						<ShowPicture styledImage={item.image}></ShowPicture>
					</section>
				)
```